pkgs/applications/misc/ulauncher/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv, fetchurl, python27Packages,
+  gnome3, gobject-introspection, wrapGAppsHook, gtk3, webkitgtk,
+  libnotify, keybinder3, libappindicator, intltool, wmctrl }:
+
+python27Packages.buildPythonApplication rec  {
+  pname = "ulauncher";
+
+  versionNumber = "4.3.2";
+  version = "${versionNumber}.r8";
+
+  src = fetchurl {
```

Comment:
This archive won't be stable because github archives are generated.

So you should use `fetchFromGitHub`.

Reply:
This is actually not a generated URL but an uploaded artefact.

Reply:
Either way, the most recent commit uses `fetchFromGithub`; the code is cleaner that way